src/Cake.Svn/Info/SvnInfoResult.cs

```diff
@@ -31,7 +31,6 @@ public sealed class SvnInfoResult
             SvnKind nodeKind)
         {
             repositoryRoot.NotNull(nameof(repositoryRoot));
-            lastChangedAuthor.NotNull(nameof(lastChangedAuthor));
```

Comment:
Why?

Reply:
@pascalberger because the returned result for lastChangedAuthor can be null. Would it better pass string.Empty if null?